src/oop/static_modifier/hard/q6/QuestionText.md

```diff
@@ -0,0 +1,39 @@
+# oop-static_modifier-hard-q6
+
+- Question Type: Java question
+
```

Comment:
Animals are an important part of farm life and require a lot of food. Write a Java class called `Farm` to keep track of the animals and available food using the requirements below. Then write a class called `TestFeeding` to test the `Farm` using the next set of requirements below.

Reply:
Add this as question text